ed_sensor_integration/tools/segmenter.cpp

```diff
@@ -240,6 +203,12 @@ int main(int argc, char **argv)
                 i_snapshot = snapshots.size();
                 snapshots.push_back(Snapshot());
                 Snapshot& snapshot = snapshots.back();
+                snapshot.world_model = world_model;
+                if (e)
+                    snapshot.area_description = std::string("on_top_of ").append(entity_id);
+                else
+                    snapshot.area_description = "";
+                std::cout << "area description" << snapshot.area_description << std::endl;
```

Comment:
```suggestion
                snapshot.world_model = world_model;
                if (e)
                    snapshot.area_description = std::string("on_top_of ").append(entity_id);
                else
                    snapshot.area_description = "";

                std::cout << "area description" << snapshot.area_description << std::endl;
```
Better to see what is in the `else` and what not.